### PR TITLE
Sfja 53

### PR DESCRIPTION
**Pull request by Antranduc**
Finished studentsDOM/formsDOM and tests

### CODE REVIEW

File: api/tests.py [hunk 1]
```diff
@@ -1,14 +1,101 @@
...
+	print('studentsDOM createStudent: ' + str(testStudentCreateStudent()))
+	print('studentsDOM deleteStudent: ' + str(testStudentDeleteStudent()))
+	print('studentsDOM removeForm: ' + str(testStudentRemoveForm()))
+	print('-------------------FORMS-------------------')
+	print('formsDOM createForm: ' + str(testFormCreateForm()))
+	print('formsDOM getInfo: ' + str(testFormGetInfo()))
+	print('formsDOM getFormData: ' + str(testFormGetFormData()))
+	print('formsDOM deleteForm: ' + str(testFormDeleteForm()))
+	print('formsDOM updateForm: ' + str(testFormUpdateFormData()))
 	return 1
```
Comment (victoriatran) [inline api/tests.py]: Do you need to return 1 here?

File: api/database/FormsDOM.py [hunk 2]
```diff
@@ -4,5 +4,41 @@
 
 app = Flask(__name__)
 MONGO_URL = os.environ.get('MONGODB_URI')
-app.config["MONGO_URI"] = MONGO_URL
+app.config["MONGO_URI"] = "mongodb://localhost:27017/sfja"
 mongo = PyMongo(app)
+
+# Creates new form.
```
Comment (victoriatran) [inline api/database/FormsDOM.py]: This is a nit pick, but if your function name is clear enough in what it does, you don't need a comment that restates it.

**Comment by chanlawrencet** [comment]
Good notes @victoriatran . @Antranduc just address these and rerequest Victoria's review!

**Comment by Antranduc** [comment]
Made a new commit, addressing the test case problem.

**Pull request merged**